src/main/java/com/instamojo/wrapper/util/HttpUtils.java

```diff
@@ -31,10 +28,34 @@
 
     private static final Logger LOGGER = Logger.getLogger(HttpUtils.class.getName());
 
+    private static String userAgent = "";
+
+    static {
+        String osInfo = System.getProperty("os.name") + "/" + System.getProperty("os.version");
+        String langInfo = "java/" + System.getProperty("java.version");
+
+        String libraryVersion = "lib-ver-not-found";
+
+        try (InputStream input = new FileInputStream("gradle.properties")) {
```

Comment:
I don't think it's a good idea to read build system files here. Are these files going to be present in the environment where this code is going to run?